config/env.example

```diff
@@ -155,3 +155,7 @@ CLAMAV_ENABLED="disabled"
 
 # Siret number used for API Entreprise, by default we use SIRET from dinum
 API_ENTREPRISE_DEFAULT_SIRET="put_your_own_siret"
+
+# Date from which email validation requires a TLD in email adresses.
+# Records (users, …) created before this date won't be affected. See #9978
```

Comment:
```suggestion
# Records (users, …) created before this date won't be affected. See #9978
# This change had been introduced by : cc53946d221d6f64c365ad6c6c4c544802eb94b4
# To set a date, we recommend using the day after you have released this commit (otherwise the app will try to validate users.email created before the feature was in prod -> might cause validation issues)
```

Reply:
thx (j'ai juste raccourci l'explication)